charts/omada-controller/values.yaml

```
@@ -0,0 +1,33 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/charts/tree/master/charts/common/values.yaml
+#
+
+image:
+  repository: mbentley/omada-controller
+  pullPolicy: IfNotPresent
+  tag: 4.2
+
+strategy:
+  type: Recreate
+
+# See more environment variables in the omada-controller documentation
+# https://github.com/mbentley/docker-omada-controller
+env: {}
+  # TZ:
+
+service:
+  port:
+    port: 8043
+
+ingress:
+  enabled: false
+
+persistence:
+  data:
+    enabled: true
+    emptyDir: false
+    mountPath: /data
+    accessMode: ReadWriteOnce
```

```suggestion
persistence:
  data:
    enabled: false
    emptyDir: false
```

This should be sufficient, and will also help the chart pass CI